plugins/bigquery/dbt/include/bigquery/macros/adapters.sql

```diff
@@ -34,6 +48,7 @@
   create or replace table {{ relation }}
   {{ partition_by(raw_partition_by) }}
   {{ cluster_by(raw_cluster_by) }}
+  {{ table_docs() }}
```

Comment:
it looks like you'll want `table_options` here right?

Reply:
Yes you're right, changing now